library/src/image.cxx

```diff
@@ -36,11 +36,17 @@
 
 namespace f3d
 {
-class image::internals
+struct image::internals
 {
   inline static const std::string metadataKeyPrefix = "f3d:";
+  // clang-format off
+  inline static const std::map<SaveFormat, std::string> saveFormatString = { 
+    { SaveFormat::PNG, "PNG" },
+    { SaveFormat::BMP, "BMP" }, 
+    { SaveFormat::JPG, "JPG" }, 
+    { SaveFormat::TIF, "TIF" } };
+  // clang-format on
```

Comment:
Would it be enough to add a trailing comma instead of disabling the formatting?
```suggestion
  inline static const std::map<SaveFormat, std::string> saveFormatString = { 
    { SaveFormat::PNG, "PNG" },
    { SaveFormat::BMP, "BMP" },
    { SaveFormat::JPG, "JPG" },
    { SaveFormat::TIF, "TIF" },
  };
```

Reply:
indeed!